app/scripts/modules/amazon/src/instance/awsInstanceType.service.spec.js

```diff
@@ -133,4 +133,61 @@ describe('Service: InstanceType', function () {
       ]);
     });
   });
+
+  describe('isBurstingSupported', function () {
+    it('identifies burstable performance instance types correctly', function () {
+      let types = ['t2.small', 't3.nano', 't3a.medium', 't4g.large', 'm5.small', 'r5.4xlarge'];
+      let service = this.awsInstanceTypeService;
```

Comment:
These should be `const` since they are not reassigned. `supportedInstanceTypes` is fine, since you are updating that value in thin the for loop.

Same goes for the rest of this file